package.json

```diff
@@ -67,6 +71,9 @@
     "postcss-discard-empty": "4.0.1",
     "postcss-import": "12.0.1",
     "postcss-loader": "3.0.0",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
```

Comment:
This is needed because the example uses react and react-dom, I'm changing this to use `my-react` and `my-react-dom`.

`parcel` is a bundler, it does not have a dependency on `react` nor on `react-dom` ;).

Reply:
I tried to change in the example to `react` and `react-dom` but since dependencies use `import React from 'react'` in their code,  it does not work. IMHO, this is not a big problem if developers have duplicates in their `node_modules`. Do you think it is a problem if there is a duplicate ?